tests/ui/components/results/paginationcomponent.js

```diff
@@ -144,3 +147,48 @@ describe('rendering the page numbers', () => {
     expect(paginate).toHaveBeenCalledWith(2, 1, 5);
   });
 });
+
+describe('properly interacts with storage', () => {
+  let defaultConfig;
+  beforeEach(() => {
+    const bodyEl = DOM.query('body');
+    DOM.empty(bodyEl);
+    DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+
+    defaultConfig = {
+      container: '#test-component',
+      verticalKey: 'verticalKey'
+    };
+  });
+
+  it('the limit property is set to the global storage search-config.limit after being created', () => {
+    const component = COMPONENT_MANAGER.create('Pagination', {
+      ...defaultConfig
+    });
+
+    const limit = component._limit;
+
+    expect(limit).toEqual(5);
+  });
+
+  it('updating the page sets global storage searchOffset', () => {
+    const component = COMPONENT_MANAGER.create('Pagination', {
+      ...defaultConfig
+    });
+
+    component.updatePage(5);
```

Comment:
if possible can we do this through the component ui with simulated clicks

Reply:
sure